docs/node-operators/how-to/upgrade-network.md

```diff
@@ -127,26 +125,42 @@ name = "v0.68.2"
 
 Check the following parameters.
 
-- `name` - This must match to the  folder name created in `step 3a`
+- `name` - This must match the created directory
 - `vega.binary.path` - It may be an absolute path or a relative (to config folder) path. Change that path when you have unzipped the new binary into a different folder.
 - `--nodewallet-passphrase-file` flag - Check if the path is correct for your node wallet passphrase.
 - `vega.rpc.socketPath` - Make sure the path to the Vega Unix sock is correct and matches the one in the Vega config.
 
-Once you have updated your Visor run config and the network config, you have to wait for the upgrade block. Visor will listen for the upgrade event and restart your binaries. 
+Once you have performed steps `2.1 Prepare network configuration (all)` and `2.2 Prepare Visor configuration (visor only)`, you don't have to do anything else. The visor will automatically restart the node once the `core` and the `data-node` (if you run one) report they are ready for Protocol Upgrade.
+
+## 2.3 Prepare for an upgrade without Vega Visor (non visor only)
 
-### 3b. Prepare upgrade if not running Visor
-You should do this step only if you are not running Visor.
+You should perform these steps only if you are NOT running Vega Visor.
 
 1. Download the new version of the Vega binary from the [releases page ↗](https://github.com/vegaprotocol/vega/releases)
 2. Unzip the downloaded binary into your file system.
 3. Update your systemd (or any other process manager) to use the new binary.
 4. Reload a systemd service: `systemctl daemon-reload`, or use your own preferred.
-5. Wait for the `upgrade block`, then restart the network with the new binary and config after the `upgrade block` is present.
 
-#### Check if Vega core is ready for restart
-To see when your Vega core node is ready for protocol upgrade, check the Vega-core logs.
+Now you are ready for `Protocol Upgrade`. Please read `3. Execute an upgrade at the agreed block (non visor only)` to know what to do next.
+
+# 3. Execute an upgrade at the agreed block (non visor only)
+
+You should perform these steps only if you are NOT running Vega Visor. If you use Vega Visor, it will perform these steps for you automatically.
```

Comment:
```suggestion
You should perform these steps only if you are NOT running Visor. If you use Visor, it will perform these steps for you automatically.
```